internal/log/log.go

```diff
@@ -125,6 +125,16 @@ func GetLevel() Level {
 	return level
 }
 
+// Log writes the given message to the logger at the given level.
+func Log(lvl Level, msg string) {
+	switch lvl {
+	case LevelDebug:
+		Debug("%s", msg)
+	case LevelWarn:
+		Warn("%s", msg)
+	}
+}
+
```

Comment:
This looks a little fishy to me as-is:

1. There's no default case in this switch; and if `lvl` is neither` LevelDebug` nor `LevelWarn`, the message is flat out ignored
2. I'd have somehow expected the other functions (`Debug`, `Info`, etc...) to be implemented by calling this one... and this one to call `printMsg`.
	- This could imply a signature like `func Log(lvl Level, fmt string, args []any)` (intentionally not using a vararg here to trick the `vet` check)

Reply:
I understand, but there aren't other `Level` values defined and I don't really want to change `Warn` semantics by adding an `Error` level:

```go
const (
	// LevelDebug represents debug level messages.
	LevelDebug Level = iota
	// LevelWarn represents warning and errors.
	LevelWarn
)
```

That's why I went with that implementation. But on second review, I only used it in a single place. I've removed it to simplify.